### PR TITLE
doc/General documentation edits

### PR DESCRIPTION
**Pull request by PipKat**
@plule-ansys I changed from "Integration Guide" to simply "Integration" because it seemed odd to call something a guide that was a single topic. I wasn't sure if it was possible to link from the README.rst file to the Integration topic since this file is repurposed. If it can be made a link and you know how, I'd love it to work that way. I'll leave some other questions/comments for you in comments. Thanks!

### CODE REVIEW

File: doc/source/integration.rst [hunk 1]
```diff
@@ -112,37 +111,39 @@ Typically, the resulting code will look like:
...
        if self.instance is not None:
            self.instance.delete()
 
-Note that it is the responsibility of the PIM to clean-up any resource and
-process associated with the product. It's however fine to double down with any
-relevant product specific clean-up.
+.. note::
+   While it is PyPIM's responsibility to clean up any resource and
+   process associated with the product, relevant product-specific cleanup
+   is still advised.
```
Comment (PipKat) [inline doc/source/integration.rst:125]: I was trying to avoid the "down down" slang.

File: src/ansys/platform/instancemanagement/__init__.py [hunk 2]
```diff
@@ -50,7 +50,7 @@ def is_configured() -> bool:
     Returns
     -------
     bool
-       ``True`` when the environment is configured to use PyPIM, ``False`` when it is not.
+        ``True`` when the environment is configured to use PyPIM, ``False`` otherwise.
```
Comment (PipKat) [inline src/ansys/platform/instancemanagement/__init__.py:53]: I'm hoping that this missing space is what is causing the odd rendering in the documentation.

File: src/ansys/platform/instancemanagement/definition.py [hunk 3]
```diff
@@ -56,8 +56,8 @@ def product_version(self) -> str:
     def available_service_names(self) -> Sequence[str]:
         """List of the available service names.
 
-        If the product exposes a gRPC API, the service will be named "grpc".
-        If the product exposes a REST-like API, the service will be named "http".
+        If the product exposes a gRPC API, the service name begins with "grpc".
```
Comment (PipKat) [inline src/ansys/platform/instancemanagement/definition.py:59]: I think my changes are correct, assuming the name given is like in Aeneid.

**Comment by PipKat** [comment]
@plule-ansys Did you want me to add PyPIM to the list of public repositories listed here: https://docs.pyansys.com/? That was my understanding based on Tuesday's team meeting.

**Review by plule-ansys — CHANGES_REQUESTED** [review]
Thanks for all the improvements

File: src/ansys/platform/instancemanagement/instance.py [hunk 4]
```diff
@@ -84,8 +84,8 @@ def services(self) -> Mapping[str, Service]:
         """List of entry points exposed by the instance.
 
         This property is only filled when the instance is ready.
-        If the instance exposes a gRPC API, it is named ``grpc``.
-        If the instance exposes a REST-like API, it is named ``http``.
+        If the instance exposes a gRPC API, the name begins with "grpc".
```
Comment (PipKat) [inline src/ansys/platform/instancemanagement/instance.py:87]:
```suggestion
        If the instance exposes a gRPC API, it is named "grpc".
```

File: src/ansys/platform/instancemanagement/instance.py [hunk 5]
```diff
@@ -84,8 +84,8 @@ def services(self) -> Mapping[str, Service]:
...
+        If the instance exposes a REST-like API, the name begins with "http".
```
Comment (PipKat) [inline src/ansys/platform/instancemanagement/instance.py:88]:
```suggestion
        If the instance exposes a REST-like API, it is named "http".
```

File: src/ansys/platform/instancemanagement/definition.py [hunk 6]
```diff
@@ -56,8 +56,8 @@ def product_version(self) -> str:
...
+        If the product exposes a REST-like API, the service name begins with "http".
```
Comment (PipKat) [inline src/ansys/platform/instancemanagement/definition.py:60]:
```suggestion
        If the product exposes a REST-like API, the service is named "http".
```

File: src/ansys/platform/instancemanagement/definition.py (hunk 3 above)
Comment (PipKat) [inline src/ansys/platform/instancemanagement/definition.py:59]:
```suggestion
        If the product exposes a gRPC API, the service is named "grpc".
```

File: README.rst [hunk 7]
```diff
@@ -131,10 +131,10 @@ Integration
 -----------
 
 PyPIM can be integrated in PyAnsys libraries to transparently switch to a remote
-instance in a suitable environment. This process is described in the integration
-guide.
+instance in a suitable environment. This process is described in the documentation's
+:ref:`integration topic`<integration>.
```
Comment (PipKat) [inline README.rst:135]:
```suggestion
:ref:`integration` topic.
```

File: README.rst [hunk 8]
```diff
@@ -131,10 +131,10 @@ Integration
 -----------
 
 PyPIM can be integrated in PyAnsys libraries to transparently switch to a remote
-instance in a suitable environment. This process is described in the integration
-guide.
+instance in a suitable environment. This process is described in the documentation's
+:ref:`integration` topic.
```
Comment (PipKat) [inline README.rst:135]:
```suggestion
instance in a suitable environment. This process is described in the
:ref:`integration` topic.
```

File: README.rst [hunk 9]
```diff
@@ -131,10 +131,10 @@ Integration
 -----------
 
 PyPIM can be integrated in PyAnsys libraries to transparently switch to a remote
-instance in a suitable environment. This process is described in the integration
-guide.
+instance in a suitable environment. This process is described in the
+:ref:`integration` topic.
```
Comment (plule-ansys) [inline README.rst:135]:
```suggestion
integration topic.
```

Unfortunately this syntax is not valid for pypi, only for sphinx, and this file is used for both. Removing the link

**Review by plule-ansys — APPROVED** [review]

**Pull request merged**